doc/nrf/device_guides/working_with_nrf/nrf53/features_nrf53.rst

```diff
@@ -0,0 +1,69 @@
+.. _nrf53_features:
+
+Features of nRF53 Series
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF53 Series System-on-Chips (SoC) integrate a high-performance Arm® Cortex®-M33 dual-core processor with a 2.4 GHz RF transceiver, in addition to advanced security features.
+All nRF53 Series SoC support Bluetooth® 5.1, Bluetooth Mesh, in addition to multiprotocol capabilities.
+
+For additional information, refer to the following resources:
+
+* `nRF53 Series`_ for the technical documentation on the nRF53 Series chips and associated kits.
+* :ref:`ug_nrf5340_gs` for getting started with the nRF5340 DK
+* :ref:`ug_nrf5340` for advanced topics specific to the nRF5340 DK
+* :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
+
+For the Thing:53, see the following user guides:
+
+* :ref:`ug_thingy53_gs` getting started with the Thingy:53
+* :ref:`ug_thingy53` for advanced topics specific to the Thingy:53
+
+Dual-core architecture
+**********************
+
+The nRF53 series introduces a dual-core Arm Cortex-M33 architecture, consisting of:
+
+* An application corr for high-performance application execution.
+* A dedicated network core optimized for wireless communication efficiency.
+
+The dual-core architecture allows for concurrent execution of application logic and wireless communication tasks.
+This ensures application processes do not interfere with network operations, enabling more responsive and reliable IoT devices.
+With one core dedicated to application tasks and the other to connectivity, devices can benefit from optimized performance in both areas.
+
+The separation of application and network functionalities simplifies firmware updates, allowing modifications on one processor without affecting the other.
+
+This architecture is ideal for devices requiring significant data processing alongside continuous wireless connectivity.
+Additionally, :ref:`ug_nrf5340_intro_inter_core` is possible between the two cores using Interprocessor Communication (IPC).
```

Comment:
This sentence looks a bit off when built because the two terms are very similar to each other:
![image](https://github.com/nrfconnect/sdk-nrf/assets/83582261/93556856-3b08-4660-889c-d598589473b8)
Could maybe mention OpenAMP before IPC to break it up a bit?

Also, looking at the Developing with nRF5340 DK page, it seems like some of the information (including the information on the cores) could maybe be moved to this new page instead?

Reply:
Ah, okay thanks for spotting - will take a look

Reply:
I noticed this before, and I think you are right - my initial concern was that some of the content in Developing with... includes links and info specifically about building for nRF5340, but I will try to see again what info can be moved from that guide to this page.

Also. do you know if OpenAMP applies to Thingy:53 as well?

Reply:
I would presume so, seeing as Thingy:53 is based on the nRF5340 SoC, but am not 100% sure on that.